.github/meta_issue_template/release-1.3.0-plugin-issues.md

```diff
@@ -0,0 +1,55 @@
+This is a component issue for release 1.3.0.
```

Comment:
Did you mean to include this?  We are still waiting on the retro feedback for 1.2.0

Reply:
We can leave it here and update later.

Reply:
It is basically a duplicate of `release-1.1.1-plugin-issues.md` I don't see a point in reviewing it, if we are going to copy/paste on-top of it, might as well do it later.